.github/workflows/generate.yaml

```diff
@@ -0,0 +1,31 @@
+---
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch: {}
+
+env:
+  GO_VERSION: "^1.16"
+
+jobs:
+  fields:
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: "actions/setup-go@v2"
+        with:
+          go-version: "${{ env.GO_VERSION }}"
+
+      - uses: "actions/checkout@v2"
+      - uses: "actions/cache@v2"
+        with:
+          path: "~/go/pkg/mod"
+          key: "${{ runner.os }}-gomod-${{ hashFiles('**/go.sum') }}"
+          restore-keys: |
+            ${{ runner.os }}-gomod-
+
+      # TODO: Automatically merge the PR if tests pass.
```

Comment:
I think this requires a Personal Access Token. Otherwise, `GITHUB_TOKEN` will be used and won't trigger subsequent workflows.